### PR TITLE
Ignoring errors when we can't assume the role

### PR DESCRIPTION
**Pull request by caiusb**
This will prevent fluentd from crashing alltogether

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.3.2
\ No newline at end of file
+1.3.2-etleap
```
Comment (romming) [inline VERSION:1]: Convention is to make the branch name 1.3.2-etleap and the version 1.3.2.0-etleap - that way we can continue to evolve without interfering with the upstream version number.

File: lib/fluent/plugin/out_s3.rb [hunk 2]
```diff
@@ -530,6 +532,9 @@ def setup_credentials
         # See http://docs.aws.amazon.com/sdkforruby/api/Aws/S3/Client.html
       end
       options
+    rescue Aws::STS::Errors::AccessDenied => e
+      log.error "Failed to assume the role #{:role_arn}, because: #{e.message}"
+      options ## return the options, this was try and use the default credentials
```
Comment (romming) [inline lib/fluent/plugin/out_s3.rb:537]: was -> will (?)

**Review by romming — APPROVED** [review]
Nice! Could you comment on whether tests pass (or configure circle build)?

**Comment by caiusb** [comment]
Closing, to open a new one on the correct branch

**Pull request closed without merging**